cmd/vicadmin/logs.go

```diff
@@ -255,10 +255,14 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 			log.Warningf("error reading %s(%s): %s\n", name, r, err)
 			continue
 		}
+		var sz int64
```

Comment:
if e is nil, sz will be 0. If e is nil then e.Size() won't work so we work with 0 instead.